.circleci/config.yml

```diff
@@ -29,6 +29,7 @@ commands:
     steps:
       - run: curl -OL https://raw.githubusercontent.com/graknlabs/build-tools/master/ci/install-bazel-mac.sh
       - run: bash ./install-bazel-mac.sh && rm ./install-bazel-mac.sh
+      - run: brew cask install homebrew/cask-versions/adoptopenjdk8
```

Comment:
why do we need this @vmax ? if this is always needed for mac, we should bundle it into `./install-bazel-mac.sh`

Reply:
@lolski mac builds will fail without JDK installed (JRE only), see https://github.com/bazelbuild/bazel/issues/7304. I'll add it to `install-bazel-mac` then.

Reply:
@lolski addressed in https://github.com/graknlabs/grakn/pull/5215/commits/2e599edfd186b9bfcf4865d6a3e1bb880f4f9886